src/lib/utils/hooks/useGetEntityHeaders.ts

```diff
@@ -1,26 +1,27 @@
 import { useEffect, useState } from 'react'
-import { EntityHeader} from '../synapseTypes'
+import { EntityHeader } from '../synapseTypes'
 import { getAllEntityHeader } from '../SynapseClient'
 import { SynapseConstants } from '..'
 import { without, chunk } from 'lodash-es'
 
-export type UseGetProfilesProps = {
-  references: string[]
+export type UseGetEntityHeaderProps = {
```

Comment:
could you revert this change as well?

export type UseGetProfilesProps = {
  references: string[]
  token?: string
}